slides/_comments_docstrings.qmd

```diff
@@ -180,84 +176,6 @@ def calculate_gyroradius(mass, v_perp, charge, B, gamma=None):
 ::::
 
 
```

Comment:
I feel like commenting out with a `<!-- -->` could be preferable so the content is not 'lost' as you do below.

Reply:
OK, happy to do that -- I thought I'd remove this one fully, as it it deprecated and therefore will probably not be taught in future

Reply:
done